main.py

```diff
@@ -139,15 +141,19 @@ def request_rapidapi_indeed_data() -> List[Opportunity]:
     rapid_jobs = []
     response = requests.get(url, headers=headers).json()
 
-    command_line_value = utils.extract_command_value()  # Extracts command-line value
+    command_line_value = utils.extract_command_value().days_needed
```

Comment:
Use a more specific variable name for the command line value. Such as `dayes_needed`.

Reply:
Resolved [here](https://github.com/acmcsufoss/acmOpportunities/pull/19/commits/7751f596eb159fd801dcf7991a5e09e4e30c69c0)